pkg/qemu/imgutil/imgutil.go

```diff
@@ -15,6 +15,18 @@ type Info struct {
 	VSize  int64  `json:"virtual-size,omitempty"`
 }
 
+func QcowToRaw(source string, dest string) error {
```

Comment:
Given that QCOW is an acronym (QEMU Copy On Write), shouldn't this be `QCOWToRaw`?

Reply:
Done